source/elens.f90

```diff
@@ -7,59 +7,86 @@ module elens
   use floatPrecision
   use crcoall
   use mod_alloc
-  use numerical_constants, only : zero
-
+  use numerical_constants, only : zero, one
+  use physical_constants, only: pmae
   implicit none
 
```

Comment:
Module variables should be private by default. Either by setting private initiallu and change the ones that need to be public, or explicitly set all variables to private/public.